### ISSUE TITLE
Investigate keyboard testing - tab component 

### ISSUE THREAD

**Issue by line47** [CMSgov/design-system#400]
Current guidance on web aim https://webaim.org/techniques/keyboard/

> **Tab** once to navigate into the group of tabs and once to navigate out of the group of tabs. **up/down** arrows or **left/right** arrows to move through tabs.

From @1Copenut 
> There's some debate on this one, about how Tabs should behave. We came down on the side of using ARIA attributes, and making tabs proper links that don't respond to arrow keys. Used this as our guidance: https://web.archive.org/web/20180724032341/https://simplyaccessible.com/article/danger-aria-tabs/

**Comment by davidakennedy** [comment]
I explored this today. Overall, I like where we're currently at with tabs. We're essentially following this [guidance from the WAI-ARIA Authoring Practices](https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html).

I did find a few spots we could improve the functionality though when you look at the guidance for keyboard interactions on that page:

- If you're hitting the right arrow in the tabs, you can get to the disabled one. That shouldn't happen.
- If you're using the left and right arrows to navigate, they should cycle through. Meaning if you reach the last tab and press the right arrow, it should go to the first tab. If you're on the first tab and press the left arrow, it should go to the last tab.

Let's track these in this issue, rather than opening a new one.

**Comment by forrestbaer** [comment]
As this issue is somewhat general, and we currently have tickets in our queue for looking at keyboard bugs/changes, I am going to close this issue here. For specific keyboard changes needing to be requested please follow up with either a new issue or a jira ticket that we can prioritize. Thanks!
